packages/@office-iss/react-native-win32/src/Libraries/Animated/NativeAnimatedHelper.win32.js

```diff
@@ -0,0 +1,593 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
is this the correct header for us to use?

Reply:
Not sure, it looks like it's a mix between which is used. I think keeping the Meta copyright makes sense since all the code is theirs and I only deleted a few lines, but not entirely sure what the guidance is. Maybe @TatianaKapos would know?

Reply:
On the macOS side I've wondered this and elected to keep the Meta headers for the sake of reducing diffs. We have to mark every diff we make so doing that kind of edit would.. suck.

Reply:
Guidance is to use the Meta header but mark specific Windows changes with comments ( see https://github.com/microsoft/react-native-windows/blob/main/vnext/src/index.windows.js#L53 as an example)